mite_selenium/__init__.py

```diff
@@ -55,30 +55,43 @@ def stop(self):
 def send_selenium_times(context):
     """Maybe would probably need more error handling
     if somebody tried to use with an incompatible browser"""
-    if context.browser.capabilities['browserName'] != 'chrome':
+    if context.browser.capabilities["browserName"] != "chrome":
         logger.warning(
-            'Timing metrics in %s are not currently supported',
-            context.browser.capabilities['browserName'],
+            "Timing metrics in %s are not currently supported",
+            context.browser.capabilities["browserName"],
         )
         return
 
     timings = context.browser.execute_script(
         'return performance.getEntriesByType("navigation")'
     )[0]
+    request_response_time = timings["responseEnd"] - timings["requestStart"]
+    first_contentful_paint = context.browser.execute_script(
+        'return performance.getEntriesByName("first-contentful-paint")'
+    )[0]["startTime"]
```

Comment:
Are you sure this works?  I've tried it in the browser console, and it doesn't seem to return anything (neither for FF nor chrome).  https://web.dev/fcp/ seems to suggest a slightly more complicated arrangement is needed to get the info (need to register a performance event watcher for paint events, which otherwise would not be recorded, which makes sense in a way -- why would the browser capture high-density perf events on the off chance that someone might want them later).  See also [this lib function](https://github.com/GoogleChrome/web-vitals/blob/master/src/getFCP.ts).

(But if it wasn't working I guess we'd get KeyErrors or something...mostly I'm curious to see what metrics are returned here, in order to analyze them like I did for the navigation ones...)